R/later.R

```diff
@@ -104,3 +104,14 @@ run_now <- function() {
 loop_empty <- function() {
   idle()
 }
+
+#' Relative time to next scheduled operation
+#'
+#' Returns the duration between now and the earliest operation that is currently
+#' scheduled, in seconds. If the operation is in the past, the value will be
+#' negative. If no operation is currently scheduled, the value will be `Inf`.
+#'
+#' @export
+next_op_secs <- function() {
```

Comment:
Is it necessary to have a wrapper function, instead of just naming the C++ function `next_op_secs`?

Reply:
Oh, I guess that works? And just write the roxygen in the .cpp file?

Reply:
Yup, that's what i had in mind.